### PR TITLE
Help for ScreenBuilder commands

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: docs/man_pages/screenbuilder/add-dataProvider.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+create
```
Comment (rosen-vladimirov) [inline docs/man_pages/screenbuilder/add-dataProvider.md]: this is the name of the command and it is used as a title of the page. You should update all your markdowns to have correct title

File: docs/man_pages/screenbuilder/add-dataProvider.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder add-dataProvider`
+Preserves information for your custom DataProvider. After you generate a DataProvider you can reuse it in as many places as you wish.
+
+The following Data Providers are available:
+    * Telerik Backend Services
```
Comment (rosen-vladimirov) [inline docs/man_pages/screenbuilder/add-dataProvider.md]: I think these * should be at the beginning of the line:
```
The following Data Providers are available:
* Telerik Backend Services
...
```
Have you tested this page as html?

File: docs/man_pages/screenbuilder/add-field.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+create
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder add-dataProvider`
```
Comment (rosen-vladimirov) [inline docs/man_pages/screenbuilder/add-field.md]: `appbuilder add field` or `appbuilder add-field` - I'm not sure what is the correct syntax

File: docs/man_pages/screenbuilder/add-dataProvider.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+create
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder add-dataProvider`
```
Comment (rosen-vladimirov) [inline docs/man_pages/screenbuilder/add-dataProvider.md]: is this really what is expected from users? `appbuilder add-dataProvider` is awful when you have to type it on the console :crying_cat_face:

File: docs/man_pages/screenbuilder/add-signin.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+create
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder add-signin`
```
Comment (rosen-vladimirov) [inline docs/man_pages/screenbuilder/add-signin.md]: should you have one "g" at the end - `signing`? Also is the command with `-` really :sob:

**Comment by rosen-vladimirov** [comment]
Rebase is required (maybe the common lib is the problem)

File: docs/man_pages/project/creation/create.md [hunk 6]
```diff
@@ -3,13 +3,15 @@ create
 
 Usage | Syntax
 ------|-------
-General | `$ appbuilder create [<Command>]`
+General | `$ appbuilder create [<Command>] [<App name>]`
+If `<App name>` is specified, new ScreenBuilder project will be created.
```
Comment (ligaz) [inline docs/man_pages/project/creation/create.md]: The public name for Scren Builder is with space between the two words.

**Comment by rosen-vladimirov** [comment]
:+1:  after green/yellow build

**Comment by Fatme** [comment]
run ci

**Pull request merged**